doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -160,28 +160,28 @@ What's next?
 =============================================== =========================
 **For more detailed information about:**        **refer to:**
 ----------------------------------------------- -------------------------
-    platform-specific notes                     `platforms`_
+    platform-specific notes                     `doc/rst/platforms`_
     online Chapel Documentation                 `chapel.cray.com/docs`_
     example Chapel programs                     ``examples/README``
-    setting Chapel environment variables        `doc/chplenv.rst`_
-    building the compiler                       `doc/building.rst`_
-    compiling Chapel programs                   `doc/compiling.rst`_
-    executing Chapel programs                   `doc/executing.rst`_
-    debugging Chapel programs                   `doc/debugging.rst`_
-    reporting bugs                              `doc/bugs.rst`_
+    setting Chapel environment variables        `doc/rst/usingchapel/chplenv.rst`_
```

Comment:
Do you think that these should not be relative to where QUICKSTART.rst now lives?  (i.e., ``chplenv.rst`` since both files are in the same directory rather than ``doc/rst/usingchapel/chplenv.rst`` which doesn't work from this location).  Though that has the downside of requiring something like ``../platforms`` for other cases.  Alternatively, we could make this ``$CHPL_HOME/doc/rst/usingchapel/chplenv.rst`` to remove the ambiguity, though that seems a little messier.

Put another way: Note that all of the paths here were (and still are with your change) written as though QUICKSTART.rst still lives in $CHPL_HOME.  I'm guessing you'd rather not move it back up there, though?